yb_migrate/src/utils/commonVariables.go

```diff
@@ -94,3 +100,41 @@ type Issue struct {
 	Suggestion   string `json:"suggestion"`
 	GH           string `json:"GH"`
 }
+
+type DataFileDescriptor struct {
+	FileType      string           `json:"FileType"`
+	TableRowCount map[string]int64 `json:"TableRowCount"`
+	Delimiter     string           `json:"Delimiter"`
+	HasHeader     bool             `json:"HasHeader"`
+}
+
+func (dfd *DataFileDescriptor) StoreDataFileDescriptor(exportDir string) {
+	dfdFilePath := exportDir + "/metainfo/flags/dataFileDescriptor"
```

Comment:
Use file path `"/metainfo/dataFileDescriptor.json"`.

Define and use a const: `const DESCRIPTOR_PATH = "/metainfo/dataFileDescriptor.json"`.

Rename `dfdFilePath` to just `filePath`--there is no other path we're dealing with in this function.

Reply:
done